src/link-package/symlink-package.ts

```diff
@@ -27,7 +27,7 @@ export const symlinkPackage = async (
 	await symlink(
 		targetPath,
 		symlinkPath,
-		'dir',
+		'junction',
```

Comment:
Guessing this is where it's used in npm?
https://github.com/npm/cli/blob/ea0b07da149767265f11d5d77d2156e2c9f43e63/workspaces/arborist/lib/arborist/reify.js#L812